crates/frontend/src/components/experiment_form/types.rs

```diff
@@ -16,6 +16,21 @@ pub struct VariantUpdateRequest {
     pub overrides: Map<String, Value>,
 }
 
+impl From<VariantFormT> for VariantUpdateRequest {
+    fn from(value: VariantFormT) -> Self {
+        VariantUpdateRequest {
+            id: value.id,
+            overrides: Map::from_iter(value.overrides),
+        }
+    }
+}
+
+impl FromIterator<VariantFormT> for Vec<VariantUpdateRequest> {
```

Comment:
Any strum macro for this?

Reply:
Strum macros are for making it easier to work with enums and strings, don't really think they have any implementation for this usecase